platforms/taobao/res/pages/index/index.js

```diff
@@ -61,5 +51,15 @@ Page({
 		touchmoveCB && touchmoveCB(event);
 	},
 	canvasOnReady () {
+		my.createCanvas({
+			id:'GameCanvas', 
+			success(canvas){
+				$global.screencanvas = canvas;
+				$global.__cocosCallback();
+			},
```

Comment:
```suggestion
			success(canvas){
			        if (!canvas) {
    				        console.error('failed to create canvas.')
			                return;
			        }
				$global.screencanvas = canvas;
				$global.__cocosCallback();
			},
```


It's better to judge whehter we have successfully created canvas, so that next time we can locate the issue quickly.